### DISCUSSION TITLE
How to set new value to texttype attribute while creating product?

### DISCUSSION THREAD

**Discussion by g-londhe** [saleor/saleor#10031] [Q&A]
Hi,
I would like to know how to set the value to attribute having type text-type while creating a product.

Currently, I'm using the following approach.

` {
                    "id": "QXXXXXXXXOjg3",
                    "richText": "{\"time\":1653976978462,\"blocks\":[{\"id\":\"**xxxxxxxM**\",\"type\":\"paragraph\",\"data\":{\"text\":\"STRING-VALUE\"}}],\"version\":\"2.22.2\"}"
                }`

And I'm setting time, block, and version hardcoded, I would like to know how I can generate those things, especially **id** dynamic?

**Comment by timuric** [comment]
We have shipped _plain text attribute_ in the core, however, it is missing in the dashboard but would be added pretty soon. Would that cover your use case?

**Reply by g-londhe** [reply]
Yes, That'll cover it.

So the setting value to this attribute would be the same as setting attribute value to the multi-value-checkbox/dropdown or it's going to be different, can you pls elaborate?

I'm setting the value to the multi-value/dropdown checkbox as below.

`{id: "QXxxxxxxxxxx4", values: ["dropdown-value-here"]}`

**Reply by timuric** [reply]
plain text attribute would accept text string without Editor.js format

**Reply by g-londhe** [reply]
Ok, Thanks. That is helpful.
